core/summary_api.php

```diff
@@ -1363,23 +1363,31 @@ function summary_by_dates_bug_count( array $p_date_array, array $p_filter = null
 		$t_count_array['open'][$t_ix] = 0;
 		$t_count_array['close'][$t_ix] = 0;
 	}
```

Comment:
Add blank line after `}` unless followed by another `}`.  See above in the function.  Apply to changes in this function.

Reply:
Fixed